### PR TITLE
altered updateRelease API with updateReleaseResource API in updateReeaseName command

### PR DESCRIPTION
**Pull request by PranaySPatil**
Modified ReleaseServer.UpdateReleaseName to use UpdateReleaseResourceAsync instead of UpdateReleaseAsync.

### CODE REVIEW

File: src/Agent.Worker/Agent.Worker.csproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="vss-api-netcore" Version="0.5.95-private" />
+    <PackageReference Include="vss-api-netcore" Version="0.5.97-private" />
```
Comment (Lovakumar) [inline src/Agent.Worker/Agent.Worker.csproj]: @TingluoHuang please review this package update.

File: src/Misc/layoutbin/en-US/strings.json [hunk 2]
```diff
@@ -492,6 +492,7 @@
   "RMTfsVCEndpointNotFound": "Cannot find required information in the job to download the Team Foundation Version Control artifact.",
   "RMUpdateReleaseName": "Update Release Name.",
   "RMUpdateReleaseNameForRelease": "Update release name to {0} for release {1}.",
+  "RMUpdateReleaseNameForReleaseComment": "Updated name to {0} using task logging command",
   "RMUserChoseToSkipArtifactDownload": "Skipping artifact download based on the setting specified.",
```
Comment (Lovakumar) [inline src/Misc/layoutbin/en-US/strings.json]: Updating release name to {0} using task logging command

File: src/Agent.Worker/Agent.Worker.csproj (hunk 1 above)
Comment (TingluoHuang) [reply src/Agent.Worker/Agent.Worker.csproj]: you should change reference to `vss-api-netcore` in all csprojs under agent repo.

Comment (PranaySPatil) [reply src/Agent.Worker/Agent.Worker.csproj]: updated all csprojs

**Pull request closed without merging**